[DOC+] ILM min_age interpretation (#98245)
elastic/elasticsearch: docs/reference/ilm/error-handling.asciidoc

{ilm-init} subsequently attempts to re-run the step that failed. 
You can use the <<ilm-explain-lifecycle,{ilm-init} Explain API>> to monitor the progress.

[discrete]
=== Common {ilm-init} setting issues

[discrete]

> ==== `min_age` calculation time

Reviewer: ```suggestion
==== How `min_age` is calculated
```

> When setting up an <<set-up-lifecycle-policy,ILM policy>> or <<getting-started-index-lifecycle-management,Automating Rollover with ILM>>, {ilm-init}'s `min_age` calculates from rollover time fallback index creation time.

Reviewer: ```suggestion
When setting up an <<set-up-lifecycle-policy,{ilm-init} policy>> or <<getting-started-index-lifecycle-management,automating rollover with {ilm-init}>>, be aware that`min_age` can be relative to either the rollover time or the index creation time.
```

> If <<ilm-rollover,ILM Rollover>> is used, the `min_age` calculates from the the index's rollover time. This is because the <<indices-rollover-index,Rollover API>> generates a new index and the new <<indices-get-settings,Index Setting's>> `creation_date` parameter is used in the calculation. If Rollover is not used within the ILM Policy then `min_age` is calculated off the original index’s `creation_date`. 

Reviewer: ```suggestion
If you use <<ilm-rollover,{ilm-init} rollover>>, `min_age` is calculated relative to the time the index was rolled over. This is because the <<indices-rollover-index,rollover API>> generates a new index. The `creation_date` of the new index (retrievable via <<indices-get-settings>>) is used in the calculation. If you do not use rollover in the {ilm-init} policy, `min_age` is calculated relative to the `creation_date` of the original index.
```

> You can override the time an index's `min_age` calculates from with either <<ilm-settings,ILM Setting>> `index.lifecycle.origination_date` or ` index.lifecycle.parse_origination_date `. 

Reviewer: ```suggestion
You can override how `min_age` is calculated using the `index.lifecycle.origination_date` and `index.lifecycle.parse_origination_date` <<ilm-settings,{ilm-init} settings>>.
```
